src/lib/components/SelectField/SelectField.jsx

```diff
@@ -125,6 +130,15 @@ SelectField.defaultProps = {
   variant: 'outline',
 };
 
+const optionPropType = PropTypes.shape({
```

Comment:
Great idea to create definition of `option`. I would create new file `prototypes.js` and import in from there as you can share this definition with SelectField and Option then.